ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -96,7 +99,25 @@ const triggerSet: TriggerSet<Data> = {
       id: 'R2S Headmarker Cone Tankbuster',
       type: 'HeadMarker',
       netRegex: { id: headMarkerData.tankLaser, capture: true },
-      response: Responses.tankCleave(),
+      delaySeconds: (data, matches) => {
+        data.tankLaserCollect.push(matches);
+        return data.tankLaserCollect.length === 2 ? 0 : 0.1;
+      },
+      alertText: (data, _matches, output) => {
+        if (data.tankLaserCollect.length === 0)
+          return;
+
+        if (data.tankLaserCollect.some((m) => m.target === data.me)) {
```

Comment:
Per note above about storing the entire regex capture group on `data`, this would then be `.includes(data.me)`.